tests/mr/device/mr_tests.cpp

```diff
@@ -28,7 +28,9 @@ INSTANTIATE_TEST_CASE_P(ResourceTests,
                         ::testing::Values(mr_factory{"CUDA", &make_cuda},
                                           mr_factory{"Managed", &make_managed},
                                           mr_factory{"Pool", &make_pool},
-                                          mr_factory{"Binning", &make_binning}),
+                                          mr_factory{"Binning", &make_binning},
+                                          mr_factory{"Limiting", &make_limiting},
```

Comment:
Since these are just adapters, and they have tests of their functionality, I don't think we need to enable them in these tests. (We don't for existing adapters)  Coverage should be good because we test all the upstream allocators that the adapters are used with.